### PR TITLE
Feature/react native 0 71 3

### PR DESCRIPTION
**Pull request by kbetl-dlb**

### CODE REVIEW

File: example/android/app/build.gradle [hunk 1]
```diff
@@ -117,7 +81,7 @@ def jscFlavor = 'org.webkit:android-jsc:+'
  * on project.ext.react, JavaScript will not be compiled to Hermes Bytecode
  * and the benefits of using Hermes will therefore be sharply reduced.
  */
-def enableHermes = project.ext.react.get("enableHermes", false);
+//def enableHermes = project.ext.react.get("enableHermes", false);
```
[the remark this answers is not in the document]
Comment (kbetl-dlb) [reply example/android/app/build.gradle:84]: removed

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -188,8 +153,8 @@ android {
 
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    //noinspection GradleDynamicVersion
-    implementation "com.facebook.react:react-native:+"  // From node_modules
+    // The version of react-native is set by the React Native Gradle Plugin
+    implementation "com.facebook.react:react-android"  // From node_modules
```
[the remark this answers is not in the document]
Comment (kbetl-dlb) [reply example/android/app/build.gradle:157]: this package: implementation "com.facebook.react:react-native:+"  is from node_modules 
the new one is from mavenCentral. Check android/example/build.gradle

[the remark this answers is not in the document]
Comment (kbetl-dlb) [reply example/android/app/build.gradle:157]: and one more thing. After this changes (load pacakge from amavenCentral nto from ndoe_modules) android build was decreased from 7-9 minutes to 2-3 mintues

File: example/android/build.gradle [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
     }
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-      classpath "com.android.tools.build:gradle:7.3.0"
+        classpath "com.android.tools.build:gradle:7.3.1"
+        classpath("com.facebook.react:react-native-gradle-plugin")
+        classpath("de.undercouch:gradle-download-task:5.0.1")
```
[the remark this answers is not in the document]
Comment (kbetl-dlb) [reply example/android/build.gradle]: I removed this line.

**Review by mateuszglapiak-dolby — DISMISSED** [review]

**Review by mateuszglapiak-dolby — APPROVED** [review]

**Pull request merged**